tools/c7n_mailer/README.md

```diff
@@ -111,6 +111,39 @@ There is a special `to` format that specifies datadog delivery, and includes the
 - metric_name: is the name of the metrics send to DataDog
 - metric_value_tag: by default the metric value send to DataDog is `1` but if you want to use one of the tags returned in the policy you can set it with the attribute `metric_value_tag`, for example in the `test-policy.yml` the value used is the size of the EBS volume. The value must be a number and it's transformed to a float value.
 
+### Slack:
+
+The Custodian mailer now supports Slack messaging as an extension of the SQS transport method, that currently leverages email as the primary notification mechanism.
+The `mailer.yml` file now supports a `slack_token` field where the user enters the Slack token corresponding to their Slack integration:
```

Comment:
The tense in these docs isn't quite right, its all present past participle, imagine reading this in six months or a year, its not just happened, its just documenting extant functionality.